samples/tab-deeplink/nodejs/pages/ExtendedDeepLink.html

```diff
@@ -0,0 +1,217 @@
+<link rel="stylesheet" href="/style.css" />
+
+<script type="text/javascript">
+    microsoftTeams.app.initialize();
+
+    function selectPeople() {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+                if (people) {
+                    document.getElementById("Attendee").value = (people.map((p) => p.email)).join(",");
+                }
+            }).catch((error) => {
+                console.log(error);
+            });
+        });
+    }
+
+    function onCallDeepLinkButtonClick(callModalities) {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+                if (people) {
+                    if (microsoftTeams.call.isSupported()) {
+                        microsoftTeams.call.startCall({
+                            targets: people.map((p) => p.email),
+                            requestedModalities: [callModalities]
+                        })
+                            .then((result) => {
+                                console.log(result);
+                            }).catch((error) => {
+                                console.log(error);
+                            });
+                    }
+                    else {
+                        alert("Call isn't supported");
+                    }
+                }
+            }).catch((error) => {
+                console.log(error);
+            });
+        });
+    }
+
+    function onSchedulingDialogClick() {
+        var subjectInput = document.getElementById("Subject");
+        var startTimeInput = document.getElementById("StartTime");
+        var endTimeInput = document.getElementById("EndTime");
+        var contentInput = document.getElementById("Content");
+        var attendeeInput = document.getElementById("Attendee");
+
+        if (subjectInput.value.trim() !== "" && startTimeInput.value !== "" && endTimeInput.value !== "" && attendeeInput.value !== "") {
+            var meetingDetails =
+            {
+                attendees: attendeeInput.value,
+                content: contentInput.value.trim(),
+                endTime: endTimeInput.value,
+                startTime: startTimeInput.value,
+                subject: subjectInput.value.trim(),
+            };
+
+            // Currently Calendar API are not supported in Microsot Teams
+            // microsoftTeams.app.initialize().then(() => {
+            //     microsoftTeams.calendar.composeMeeting(meetingDetails).
+            //         then((result) => { console.log(result); }).
+            //         catch((error) => { console.log(error); });
+            // });
```

Comment:
We can remove this if not needed in future.

Reply:
It might be needed Wajeed told to keep it that way.